spaces/domains.json

```
@@ -5,6 +5,7 @@
   "vote.redemptiontoken.com": "redemptioncoin.eth",
   "vote.ecollective.xyz": "builders.ecollective.xyz",
   "crypto-legal-action.mycryptolawyer.xyz": "mycryptolawyer.eth",
+  "proposals.unore.io":"unoredao.eth",
```

You will need to add a space between your domain and your DNS value.
```suggestion
  "proposals.unore.io": "unoredao.eth",
```